api_loader.go

```diff
@@ -139,6 +128,29 @@ func processSpec(referenceSpec *APISpec,
 		return &chainDef
 	}
 
+	pathModified := false
+	for {
+		hash := generateDomainPath(referenceSpec.Domain, referenceSpec.Proxy.ListenPath)
+		if v, e := ListenPathMap.Get(hash); !e || v.(int) < 2 {
```

Comment:
Same as the original `val`, just an interface value. It's quite common as a single-char name, like `i` and `s`.